src/MSALWrapper/AuthFlow/AuthFlowResult.cs

```diff
@@ -24,10 +24,12 @@ public AuthFlowResult()
         /// </summary>
         /// <param name="tokenResult">A <see cref="MSALWrapper.TokenResult"/>.</param>
         /// <param name="errors">A list of errors encountered while getting (or failing to get) the given token result. Will initialize a new empty List if null is given.</param>
-        public AuthFlowResult(TokenResult tokenResult, IList<Exception> errors)
+        /// <param name="authFlowName">The name of the authflow from which the AuthFlowResult is returned or cretaed.</param>
+        public AuthFlowResult(TokenResult tokenResult, IList<Exception> errors, string authFlowName)
         {
             this.TokenResult = tokenResult;
             this.Errors = errors ?? new List<Exception>();
+            this.AuthFlowName = authFlowName;

```

Comment:
To help prevent null ref exceptions, because C#, I think we should ensure this is not null. (I acknowledge that in the future we should enable this kind of checking in the compiler since a newer version of C# supports it). But for now,

```
this.AuthFlowName = authFloweName ?? throw new ArgumentNullException(nameof(authFlowName));

Reply:
@kyle-rader understood. Updated the changes. :)